File: 217-contains-duplicate/contains-duplicate.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function(nums) {
    nums.sort();
    if(nums.length==1){
        return false
    }
    for(let i=1;i<nums.length;i++){
        if(nums[i-1]==nums[i])
            return true
    }
    return false;
};

File: 217-contains-duplicate/contains-duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        map<int,int> mp;
        for(int val:nums){

            mp[val]++;
            auto iter = mp.find(val);
            if(iter!=mp.end()&&iter->second>1)
                return true;
        }
        return false;
    }
};